[MWAN] Virtual Connector patch (#12882)
cloudflare/cloudflare-docs: content/magic-wan/configuration/connector/configure-virtual-connector.md

1. When setting up your VMware ESXi, you need to create port groups for Virtual Connector. Go to **Networking** > **Port groups**, and prepare your vSwitch port groups and/or VLANs for your desired network topology. For example, a simple deployment typically has:
    - A WAN port group where the Virtual Connector will get an IP address (static or DHCP) that has access to the Internet.
    - A LAN port group, where the Virtual Connector will act as default router, and possibly DHCP server.

>     - A "null", or "unused", port group for allocating unused virtual interfaces in the Virtual Connector. You can, for example, create a null port group with the name of `Null port group`, and a **VLAN ID** of `999`.

Reviewer: Should this be "null" and "unused" or `null` and `unused`?

> 2. Extract the files in the OVA image provided by your account team. For example:

Reviewer: Cloudflare account team? Or other account team?

    2. Allocate any unused ports to the `null` port group.
    3. Take note of your configuration. You will need this information to configure your network in the Cloudflare dashboard.
8. Before completing the deployment wizard, disable **Power on automatically**. This is important so that you can configure the license key prior to boot.
9. Configure the virtual machine with the license key your account team provided you:
    1. Select the Virtual Connector's VM > **Settings**.
    2. Go to **VM Options** > **Advanced** > **Edit Configuration**.
    3. Select **Add parameter** to add your lincence key. Scroll down to the last entry (this is where VMware adds the new parameter), and add the following two new entries:
        - **Key**: `guestinfo.cloudflare.identity`
        - **Value**: `<YOUR_LICENCE_KEY>`

  {{<Aside type="note">}} You cannot use the same license key twice, or reuse a key once the virtual machine has been registered with Cloudflare. You need a new key from your account team for every new Virtual Connector.{{</Aside>}}

10. Select **Save** to finish configuring your Virtual Connector.

> 11. Now, you need to continue set up in your Cloudflare dashboard.

Reviewer: ```suggestion
11. Continue set up in your Cloudflare dashboard in the [next section]().
```
Optional, but feels more precise in direction.
